### PR TITLE
[Bugfix] Fixed Issue Where Input Box Not Shown Up in iOS 14

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
Fixed an issue where input box not shown up in iOS 14 devices. checked with @palatter, confirmed that the version check in chat view should be removed as legacy implementation that was depended on iOS 15 is no longer being used.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test
issue only happening with iOS 14 devices, please download iOS 14 runtime to test it out
1. launch the chat view
2. check if input box is shown up

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
|  Simulator Screen Shot - iPhone SE (2nd generation) - 2023-01-04 at 11 29 53 | Simulator Screen Shot - iPhone SE (2nd generation) - 2023-01-04 at 11 29 07 |

### CODE REVIEW

**Review by palatter — APPROVED** [review]
Looks good.

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 import SwiftUI
 
-@available(iOS 15.0, *)
```
Comment (emlynmac) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift:8]: I would have thought, that this restriction and the comment in the file above points to an issue with this not working on iOS 14. 
What now happens on iOS 14?

**Review by vhuseinova-msft — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift (hunk 1 above)
Comment (palatter) [reply AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift:8]: This was left over by mistake from refactoring of the message input field. The previous version needed iOS 15

Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift:8]: From what I understand, there are used to be a legacy implementation on input box being used which used iOS 15 APIs and @palatter added this version check here. But after adding the multi-line support, the input box was re-written which no longer depends on the old APIs. And therefore, this version check is no longer needed

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**